cmd/tikv-server/src/main.rs

```diff
@@ -220,6 +220,11 @@ fn main() {
         process::exit(1)
     }
 
+    // Initialize the async-backtrace.
```

Comment:
I think it would be better to add a config item for this feature, in case it cause some performance regression or unexpected memory usage.

Reply:
And maybe we need some benchmark when the subscriber is enabled globally.